### PR TITLE
Some folks just need to cooldown

### PR DESCRIPTION
**Pull request by HarrySadleir**

### CODE REVIEW

File: src/listeners/cooldown.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {Listener} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+
+const cooldown: Listener<"messageCreate"> = {
+    event: "messageCreate",
+    procedure: async (client: Client, message: Message) => {
+        if (message.author.bot) return;
+
+        if (message.content.toLowerCase().includes("cooldown"))  {
+            const messageToSend = `Hi ${message.author.username}, it seems you have complained about about the autobot's cooldown time. We have graciously doubled your cooldown time in response, have a nice day :smiley:`;
```
Comment (braxtonhall) [inline src/listeners/cooldown.ts:10]: I think if you do `message.author.toString()` it will @ the user!

**Review by braxtonhall — CHANGES_REQUESTED** [review]
Tiny change and I will make it go live!

File: src/listeners/cooldown.ts (hunk 1 above)
Comment (HarrySadleir) [reply src/listeners/cooldown.ts]: Changed

**Review by braxtonhall — APPROVED** [review]
LGTM

**Pull request merged**